pyecharts/translator/compat.py

```diff
@@ -0,0 +1,50 @@
+# coding=utf8
+
+from __future__ import unicode_literals
+
+import inspect
+import sys
+
+try:
+    from metapensiero.pj.api import translates
```

Comment:
I think this tighten integration of javascripthon may not be good, because any new changes javacript translation require a new release of pyecharts. Should it be separate library, javascript translation can evolve on its own.